### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by dnmfarrell**

### CODE REVIEW

File: vercel.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     },
     {
       "source": "/breach",
-      "destination": "/reset",
+      "destination": "/using/id/guide-to-resets",
```
Comment (tinnus-napbus) [inline vercel.json]:
```suggestion
      "destination": "/reset",
```

Comment (tinnus-napbus) [reply vercel.json]: this was used for the page about the network breach at the end of 2020, not for info about breaching, so should probably be left as-is (although that page doesn't work for some reason, dunno why though. I think it's missing a corresponding .js file)

**Review by tinnus-napbus — CHANGES_REQUESTED** [review]

File: vercel.json (hunk 1 above)
Comment (dnmfarrell) [reply vercel.json]: Gotcha, I've undone that change.

**Review by tinnus-napbus — APPROVED** [review]
LGTM

**Pull request merged**